### ISSUE TITLE
UX Writing for site IA

### ISSUE THREAD

**Issue by Eevinjennings** [hackforla/access-the-data#156]
### Overview
Help with writing/terminology for site navigation and identify blockers.

### Action Items
- [ ] Identify which materials were used (IA site map, interactive prototype, etc.)?
- [ ] Meet with @hannahsgr and @ashleyxz for background context/questions on 4/26
- [ ] Access dataset
- [ ] Confirm next steps

### Resources/Instructions
[Wiki](https://github.com/hackforla/access-the-data/wiki)
[Tree Test Notes](https://docs.google.com/document/d/1SxJQpzL7vLbUaYyiEJfHUuvra0DbVOraJbjKLK-laEI/edit?pli=1)
[IA Tree Test Research Plan](https://docs.google.com/document/d/1oY6cs4Qp3m4e96cta7417Y5H-wM6U3yadb3TG1Eaqvg/edit)
[Figma Site map](https://www.figma.com/file/nS5IqnutUT2FuPwwJObHaY/?node-id=2%3A7)
[Figma IA](https://www.figma.com/file/nS5IqnutUT2FuPwwJObHaY/Access-the-Data-Main-Figma?node-id=2%3A9)
[Issue 143](https://github.com/hackforla/access-the-data/issues/143)

**Comment by Eevinjennings** [comment]
### Update
1. Materials used for tree test: [IA site map. ](https://www.figma.com/file/nS5IqnutUT2FuPwwJObHaY/Access-the-Data-Main-Figma?node-id=2%3A7)
2. Noted that "All content reference page" isn't part of tree.
3. Dataset shared in Slack - Destination map with counif fcn in Excel.
4. Hannah: 
> Please comment on the Google Doc I linked earlier. As for the best next steps, please go over the Google doc linked and get familiar with the tree testing issues. On Tuesday we can discuss solutions."

6. Lucy referenced issues #87 and #106 for future interest with UX Writing (will keep on backburner for now)
7. Lucy referenced #145 and said:
> "but before we dive into copy, we should do some user flows and wireflows and have you familiarize yourself with what we have and go through past usability tests with copy feedback."
8. Learned that Aubrey Schiavone joined as UX writer 
9. Learned that there is no current writing style guide/system 

### Action Items
- [ ] Examine dataset and analyses
   - [ ] Setup statistical pre-post comparisons & dataviz options
   - [ ] Read through & comment on tree test doc
- [ ] Reach out to Aubrey for possible meeting/collab
- [ ] Reach out to content team for consult about style guide
   - [ ] Create writing style/guide
- [ ] Confirm whether I need to make a new issue for lists like these or if posting them as comments is appropriate. @lrchang2 ?

### Blockers:
- Organization is a bit all over the place right now; looking forward to consolidating all of this stuff into one centralized place.
- Waiting on meeting with Ashley & Hannah tomorrow for next steps

**Comment by Eevinjennings** [comment]
### Update:
Met with Hannah and Ashley to ideate solutions for tree test results.

### Action Items:
- [ ] Run a few moderated interviews for some of the IA structure and terms. 
- [ ] A/B tests for some of the alternative terms.
- [ ] Consolidate all of the different spreadsheets/materials into a single space for ease of reference/newly onboarded members.
- [ ] Share IA update with project teams on Wednesday and figure out where to go from there.

### Blockers:
- It seems that there is a disconnect between design, data science, and IA. Hannah planned to sync up with them to confirm whether some of the nav items are even still needed.

**Comment by Eevinjennings** [comment]
Update: 
Hannah is analyzing the A/B test data. 
Blockers:
Waiting on results to see what is next.
